ash/src/extensions/khr/device_group_creation.rs

```diff
@@ -13,7 +13,13 @@ pub struct DeviceGroupCreation {
 }
 
 impl DeviceGroupCreation {
+    /// Call [`Self::new_()`] with a borrow of [`Entry`] instead.
+    #[deprecated = "Unnecessarily requires a move of `Entry`. Call `new_()` with a borrow instead."]
     pub fn new(entry: Entry, instance: &Instance) -> Self {
+        Self::new_(&entry, instance)
+    }
+
+    pub fn new_(entry: &Entry, instance: &Instance) -> Self {
```

Comment:
Is this necessary? Cloning the `Entry` is dumb, but pretty harmless.

Reply:
We'll fix it in 0.38 (#753) so yes, maybe it is annoying to force people to switch to `new_()` and then back again later. I'll drop this change, `Entry` isn't all that large though and this extension likely isn't very popular.